src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -2376,6 +2377,7 @@ MethodTableBuilder::EnumerateMethodImpls()
                 {
                     BuildMethodTableThrowException(IDS_CLASSLOAD_MI_ILLEGAL_TOKEN_BODY);
                 }
+                /* TODO: what to do about this check
```

Comment:
The spec update describes the changes here. This check should be preserved for instance virtuals, but there should be a base check for static virtual overrides.